theforeman.org/pipelines/test/kafo.groovy

```diff
@@ -11,11 +11,33 @@ pipeline {
                 axes {
                     axis {
                         name 'ruby'
-                        values '2.4', '2.5', '2.6'
```

Comment:
If we drop 2.4, then IMHO it should also be reflected in the gemspec. Also a PR to drop Puppet 5 support would be the correct thing to do.